### PR TITLE
etna materialize

### PR DESCRIPTION
**Pull request by graft**
Surprisingly easy delivery of a "materialize" command, which just involves connecting the Materialize workflow to a local Etna::Filesystem. `etna materialize --project-name my_project` results in a local folder `my_project` containing the project data.

If the Filesystem can report size information (i.e. responds to `stat`) sync_metis_data_workflow will skip a file if it exists with the same file size. Does not check hashes. Hopefully this makes resuming/retrying less painful. Records (.json) still get re-downloaded every time.

### CODE REVIEW

File: etna/lib/etna/clients/metis/workflows/sync_metis_data_workflow.rb [hunk 1]
```diff
@@ -82,20 +94,7 @@ def copy_file(dest:, url:, stub: false)
                 if start_time == end_time
                   next
                 end
-
```
Comment (coleshaw) [inline etna/lib/etna/clients/metis/workflows/sync_metis_data_workflow.rb:85]: Just a thought, but this has been useful in debugging ETLs....might be nice to put verbose logging like this behind some sort of `--debug` or `--verbose` flag in the future, if we find a need for it.

File: etna/lib/commands.rb [hunk 2]
```diff
@@ -238,6 +238,58 @@ def execute(email:, task: false, project_name: nil)
...
+      include WithEtnaClients
+
+      string_flags << "--project-name"
+      string_flags << "--log-file"
+
+      def logger
+        @logger ||= @logger = Etna::Logger.new(@log_file, 0, 1048576)
+      end
+
+      def execute(project_name:, log_file:'/dev/stdout')
```
Comment (coleshaw) [inline etna/lib/commands.rb:256]: This is awesome! I can see where this might be expanded in the future, since folks might only want `sc_seq` data for example. Not for this PR, thought.

**Review by coleshaw — CHANGES_REQUESTED** [review]
This looks great! Should we bump the gem version in `etna.gemspec` so it gets re-published? Then we can pass that info along to users.

File: etna/lib/etna/clients/metis/workflows/sync_metis_data_workflow.rb (hunk 1 above)
Comment (coleshaw) [reply etna/lib/etna/clients/metis/workflows/sync_metis_data_workflow.rb:85]: Also not necessary for this PR...

Comment (graft) [reply etna/lib/etna/clients/metis/workflows/sync_metis_data_workflow.rb:85]: It seemed hard to follow in any case, especially with concurrency turned up, because it doesn't print file names, just rates. We could just put it behind logger&.debug and let them turn up the log level.

What I wish for is a nice progress bar here if using STDOUT, but I'm not sure how to deal with concurrency in this case.

Comment (coleshaw) [reply etna/lib/etna/clients/metis/workflows/sync_metis_data_workflow.rb]: Oh good point. Perhaps would be nice to include file names along with the rate on the log line. Yeah, progress bar with concurrency seems difficult ...

**Review by coleshaw — APPROVED** [review]
